test/XRP/wallet-factory.test.ts

```diff
@@ -79,4 +79,76 @@ describe('Wallet Factory', function (): void {
     // THEN the wallet is undefined.
     assert.isUndefined(wallet)
   })
+
+  it('walletFromSeedAndDerivationPath - valid inputs - derivation path 0', function (): void {
+    // GIVEN a wallet factory, a valid seed and a valid derivation path.
+    const walletFactory = new WalletFactory(XrplNetwork.Main)
+    const seed =
+      'f5dd847bd41be2deeb5f3596ab614a95e3cebb73dcd4270df9681de9e3ee5fe34ef07beb05fea610bf2cfdffb6dc3f1593fd966ab5afd30e803af66b68df32d4'
```

Comment:
Worth saying what this seed is not hex encoded in a comment?

Reply:
@hbergren I'm a bit confused what you're looking for here. Typically, what XRP Ledger refers to as a seed is a base58check encoded string beginning with 's'.  These bytes *are* hex encoded. 

Both are 'seeds' in that they are representations of bytes which are used as a seed. 

I updated this variable name to be `seedHex` to hopefully make that clearer. If you'd like, I can run this through a base58check encoder and produce a seed in base58check form. I think generally folks who are deriving HD wallets tend to be multi-coin wallets (imagine TrustWallet or Exodus) and they'll have seeds in a generic form instead of a ripple-specific form.

Reply:
Makes sense. Very helpful explanation.